Plugins/Area/NWScript/nwnx_area.nss

```diff
@@ -288,6 +288,12 @@ int NWNX_Area_GetAmbientSoundDayVolume(object oArea);
 /// @return The volume.
 int NWNX_Area_GetAmbientSoundNightVolume(object oArea);
 
+/// @brief Create a sound object.
+/// @param lLoc The location where the sound object will be created
+/// @param sResRef The ResRef of the sound object
+/// @return The sound object.
+object NWNX_Area_CreateSoundObject(location lLoc, string sResRef);
```

Comment:
The pattern is that all NWNX_Area functions take `object oArea` as the first argument, so could you please expand this into `object NWNX_Area_CreateSoundObject(object oArea, vector vPosition, string sResRef);`?